### ISSUE TITLE
Bullets type parameter

### ISSUE THREAD

**Issue by Etienne-TR** [Zettlr/Zettlr#3903]
### Description

Bullets for item list alternate function of list level.
```
* Level 1
    - Level 2
        + Level 3
            * Level 4 etc.
```
This is pretty but this is not always wished, that should be useful to make this behavior disabled.

### Proposed Changes

An option in parameters that could be deselected. Something like this :   

Unordered list :  
- [ ] Automatic alternated bullets

### Caveats

I think there is no caveat.

### Do you Wish to Attempt Implementing this Yourself?

No

### Zettlr Version

Nightly

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Operating System Version

Ubuntu 20.04.4 LTS

**Comment by nathanlesage** [comment]
I have a better idea, but I still need time to figure this out. The main impetus here was to provide a way to automatically re-number lists. What I will do is the following:

1. Make it so that when you tab/shift-tab the list marker that gets pushed to a new list takes over what the majority of the items on that level have (i.e. it stays as it is if you create a new list level)
2. Provide a small setting where you can state how you want this to work (i.e. default `---`, dashes for all levels, that you could, if you want, change to `+-*` or similar)

I'll implement this as soon as I find a good, and easy way to do so! If you have additional cool stuff we could do with that, please add your comments here :)

**Comment by Etienne-TR** [comment]
I don't really understand. If automatically generated list marker (on tab/shift-tab/enter) is the majority one in its level (or just superior if new level), it can't follow the (for instance) `---` or `+-*` parameter. It seems to me to be an either or. Could you give again elements I don't get ?

**Comment by nathanlesage** [comment]
> It seems to me to be an either or.

Right, that is true. The first one would possibly be closer to the behavior as it has been until now, whereas the second one would be a fancy new way to ensure that different list levels are easier to distinguish because they use different list style indicators. The last thing has actually been something that has been nagging me for quite some time and I was really happy to find out that it's now possible to implement that.

**Comment by Etienne-TR** [comment]
Maybe it could be useful to add an action in the context menu when right-click on a list selection to deal with old one bullet type lists.
After `Insert an unordered list`, add something like `Update bullets from pattern` (pattern `*-+` or another from parameters).
That seems useful to you ?

**Comment by dustdfg** [comment]
> 1. Make it so that when you tab/shift-tab the list marker that gets pushed to a new list takes over what the majority of the items on that level have (i.e. it stays as it is if you create a new list level)

Just found a bug related to it I wasn't sure about separate issue...

I made ordered list that contains unordered sublist. I press enter and it creates new item of sublist. 
![screen-1703875307](https://github.com/Zettlr/Zettlr/assets/97330399/fe24639b-c902-4aac-bea1-18d3b5cdceee)

I press shift-tab to go one level above and expect to see `2.` item list but I see `*`

![screen-1703875317](https://github.com/Zettlr/Zettlr/assets/97330399/c4903332-8e17-40a8-aaf3-29e00887ceb7)
